docs/UserGuide.md

```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
 > Executing the command `note 1 /edit` will replace the contents of the command box with `note 1 Wants to gain muscle`.
 <hr>
 
+### Adding or overriding exercise(s) of clients : `fitadd`
+
+Format: `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `
```

Comment:
`n/EXERCISE_NAME` is not optional, so it should not be in `[ ]`.

Also, should the default exercise prefixes be mentioned separately?

Maybe you could have a subsection about the prefixes specifically.

Reply:
Technically speaking it is.

You must either specify a specific exercise name or use a number of default supported prefixes, but not both together. Not sure how to represent that.